### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by superzuc**
https://superzuc.github.io/layout_search-bar-airbnb/

https://superzuc.github.io/layout_search-bar-airbnb/report/html_report/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,39 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="searchBar_big_container">
+      <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="post"
+      data-qa="big"
+      >
```
Comment (polosanya) [inline src/index.html:19]:
```suggestion
      <form 
        action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
        method="post"
        data-qa="big"
      >
```

File: src/index.html [hunk 2]
```diff
@@ -11,6 +11,39 @@
...
+        <label for="searchBar_big"></label>
+        <input 
+        type="search"
+        id="searchBar_big"
+        name="searchBar"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+        >
```
Comment (polosanya) [inline src/index.html:27]:
```suggestion
        <input 
          type="search"
          id="searchBar_big"
          name="searchBar"
          placeholder="Try &quot;Los Angeles&quot;"
          data-qa="keypress"
        >
```
Fix all cases.

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+#searchBar_big:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-family: Avenir-Heavy, Arial, Helvetica, sans-serif;
+}
+
+.submit_searchButton {
+  box-sizing: border-box;
+  position: absolute;
```
Comment (polosanya) [inline src/style.css:53]: Pay attention to CSS properties order
https://9elements.com/css-rule-order/
You can also set the same box-sizing for all elements using * selector

**Review by polosanya — CHANGES_REQUESTED** [review]
Almost perfect!

**Review by polosanya — CHANGES_REQUESTED** [review]
Links don't work(
Did you make `npm run deploy`?

**Comment by superzuc** [comment]
I don't know why the link don't work and yes, i've tryed few times on npm run deploy

**Pull request closed without merging**